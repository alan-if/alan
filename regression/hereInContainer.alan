The x Isa object In cont
End The x.

The cont Isa object At l
Container
End The cont.

The l Isa location
End The l.

Start At l.
  If x HERE Then "X is here." Else "X is not here but ought to be." End If.