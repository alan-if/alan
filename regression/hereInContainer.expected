########## hereInContainer ##########

        No warnings or errors detected.


X is here.


L
There is a cont here.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
